tests/formats/pdb.cpp

```diff
@@ -232,14 +235,84 @@ TEST_CASE("Write files in PDB format") {
 }
 
 TEST_CASE("PDB files with big values") {
-    auto tmpfile = NamedTempPath(".pdb");
-    auto trajectory = Trajectory(tmpfile, 'w');
+    SECTION("Unit cell and Coordinates") {
+        auto tmpfile = NamedTempPath(".pdb");
+        auto trajectory = Trajectory(tmpfile, 'w');
+
+        auto frame = Frame(1);
+        frame.set_cell(UnitCell(1234567890));
+        CHECK_THROWS_AS(trajectory.write(frame), FormatError);
+
+        frame.set_cell(UnitCell(12));
+        frame.positions()[0] = Vector3D(123456789, 2, 3);
+        CHECK_THROWS_AS(trajectory.write(frame), FormatError);
+    }
+
+    SECTION("Default residues") {
+        auto tmpfile = "data/pdb/tmp.pdb";
```

Comment:
You can use `auto tmpfile = NamedTempPath(".pdb");` instead, it will automatically do the cleanup for you, and will create a file in /tmp instead.

Reply:
You know I tried that, and I could read the file doing:
```
    std::ifstream checking(tmpfile);
    std::string content((std::istreambuf_iterator<char>(checking)),
                         std::istreambuf_iterator<char>());
```
but, for some reason, the last lines would always be missing. And if I tried to read it through chemfiles, it would complain that the file had 0 frames in it.
I'll try again and let you know.

Reply:
You need to make sure the `Trajectory` object goes out of scope before opening it with either chemfiles or `std::ifstream`: the file is synced when the destructor runs.

Reply:
aaaahhh, I thought that the file would be deleted if `Trajectory` goes out of scope, but that happens when the `NamedTempPath` object goes out of scope. Alright, I'll fix it